src/tools/remote-test-client/src/main.rs

```diff
@@ -69,19 +72,38 @@ fn spawn_emulator(target: &str, server: &Path, tmpdir: &Path, rootfs: Option<Pat
     }
 
     // Wait for the emulator to come online
-    loop {
+    let mut total_dur = Duration::from_secs(0);
+    let timeout = env::var(CONNECT_TIMEOUT_ENV).ok().map_or(DEFAULT_CONNECT_TIMEOUT, |timeout| {
```

Comment:
Let's put this in a helper function. Let's also include the value and error in the panic message. What do you think of doing it like this?

```rs
pub fn connect_timeout() -> Duration {
    match env::var(CONNECT_TIMEOUT_ENV).ok() {
        Some(timeout) => timeout
            .parse()
            .map(Duration::from_secs)
            .unwrap_or_else(|e| {
                panic!("error: parsing `{CONNECT_TIMEOUT_ENV}` value \"{timeout}\" as seconds failed: {e}")
            }),
        None => DEFAULT_CONNECT_TIMEOUT,
    }
}
```